lstchain/reco/dl1_to_dl2.py

```diff
@@ -0,0 +1,517 @@
+"""Module with functions for Energy and disp_norm reconstruction and G/H
+separation. There are functions for raining random forest and for
+applying them to data. The RF can be saved into a file for later use.
+
+Usage:
+
+"import dl1_to_dl2"
+"""
+from typing import Dict
+
+import numpy as np
+import pandas as pd
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from sklearn.externals import joblib
+from sklearn.model_selection import train_test_split
+import os
+from . import utils
+from astropy.utils import deprecated
+from ..io import read_configuration_file
+
+
+# Standard models configurations - to be moved later in a default configuration file
+
+random_forest_regressor_args = {'max_depth': 50,
+                                'min_samples_leaf': 50,
+                                'n_jobs': 4,
+                                'n_estimators': 50,
+                                'bootstrap': True,
+                                'criterion': 'mse',
+                                'max_features': 'auto',
+                                'max_leaf_nodes': None,
+                                'min_impurity_decrease': 0.0,
+                                'min_impurity_split': None,
+                                'min_samples_split': 2,
+                                'min_weight_fraction_leaf': 0.0,
+                                'oob_score': False,
+                                'random_state': 42,
+                                'verbose': 0,
+                                'warm_start': False,
+                                }
+
+
+random_forest_classifier_args = {'max_depth': 2,
+                                 'min_samples_leaf': 10,
+                                 'n_jobs': 4,
+                                 'n_estimators': 'warn',
```

Comment:
What does the 'warn' option mean for n_estimators?
I'm getting an error here because n_estimators is not supposed to be a string: ValueError: n_estimators must be an integer, got <class 'str'>.

Reply:
Absolutely right @misabelber, my mistake.
This is fixed now.